src/common/Session.cpp

```diff
@@ -151,45 +153,59 @@ namespace SDDM {
 
         qDebug() << "Reading from" << m_fileName;
 
-        QFile file(m_fileName);
-        if (!file.open(QIODevice::ReadOnly))
-            return;
+        QSettings Settings(m_fileName, QSettings::IniFormat);
```

Comment:
style throughout sddm is to use lowerCamelCase for variables

Reply:
Ok, I changed that (will upload later) and hope that I did not forget a variable.